ament_pep257/ament_pep257/main.py

```diff
@@ -45,11 +45,14 @@ def main(argv=sys.argv[1:]):
     parser.add_argument(
         '--ignore',
         nargs='*',
-        default=[
-            'D100', 'D101', 'D102', 'D103', 'D104', 'D105', 'D106', 'D107',
-            'D203', 'D212', 'D404',
-        ],
+        default=['D1'],
         help='The pep257 categories to ignore')
+    parser.add_argument(
+        '--select',
+        nargs='*',
+        default=[],
+        help='The additional pydocstyle checks to run'
```

Comment:
Given that `--ignore` turns into using `--add-ignore` and `--select` turns into using `--add-select`, I think the terminology for the help text should be the same. `--ignore` is using the term "pep257 categories", `--select` is using "pydocstyle checks", and the [pydocstyle docs](http://www.pydocstyle.org/en/5.0.2/usage.html#command-line-interface) uses the term "error codes to check." Can we pick one and go with it, with a personal preference to upstream's term?

Reply:
fixed in 0b9856b